src/main/java/me/moodcat/eemcsdata/JsonData.java

```diff
@@ -0,0 +1,68 @@
+package me.moodcat.eemcsdata;
+
+import lombok.Data;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * A data class to store the JSON elements in.
+ */
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class JsonData {
+
+    @JsonProperty("tonal")
+    private Tonal tonal;
+
+    @Data
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    static class Tonal {
```

Comment:
This inner class is part of the public interface, as it can be returned. Therefore it should be publicly visible and be documented.